modules/nw-cudn-localnet.adoc

```diff
@@ -0,0 +1,114 @@
+//module included in the following assembly:
+//
+// * networking/multiple_networks/primary_networks/about-user-defined-networks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-cudn-localnet_{context}"]
+= Creating a ClusterUserDefinedNetwork CR for a Localnet topology
+
+A `Localnet` topology connects the secondary network to the physical underlay. This enables both east-west cluster traffic and access to services running outside the cluster. Connecting to additional services outside the cluster requires the additional configuration of the underlying Open vSwitch (OVS) system on cluster nodes.
+
+.Prerequisites
+
+* You have logged in as a user with `cluster-admin` privileges.
+
+* You have created and configured the Open vSwitch (OVS) bridge mapping to associate the logical OVN-Kubernetes network with the physical node network through the OVS bridge. For more information, see "Configuration for a localnet switched topology".
```

Comment:
@JoeAldinger Could we include another prerequisite i.e.
CUDN has been configured in primary role.

If there is not one then there will undesired results creating pods with secondary network.

Reply:
I don't follow - it should be possible to have a secondary attachment to a localnet C-UDN even if the workload is attached to the cluster default network.

@asood-rh